src/doc_handler.py

```
@@ -0,0 +1,482 @@
+        
+        attachment_info = response.json()
+        content_bytes_b64 = attachment_info.get("contentBytes", "")
+        
+        if not content_bytes_b64:
+            logger.warning(f"No content bytes found in attachment: {attachment_name}")
+            return None
+        
+        # Decode base64 content
+        try:
+            file_bytes = base64.b64decode(content_bytes_b64)
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)